libraries/c_sdk/standard/mqtt/README.md

```diff
@@ -1,18 +1,46 @@
-# MQTT Shim for MQTT V2.x.x APIs
+# MQTT Compatibility Layer for MQTT V2.x.x APIs
 
-This MQTT shim library supports backward compatibility for the MQTT V2.x.x APIs using the [coreMQTT library](../../../coreMQTT/README.md). All the [APIs in MQTT v2.x.x](include/iot_mqtt.h) are supported in this shim implementation. 
-This shim will be supported only as a short term solution to provide backward compatibility. We recommend using the redesigned [coreMQTT library](../../../coreMQTT/README.md) for applications requiring long term support.
+MQTT Compatibility Layer for MQTT V2.x.x APIs, is using the [coreMQTT library](https://github.com/FreeRTOS/coreMQTT/blob/master/README.md). All the [APIs in MQTT v2.x.x](include/iot_mqtt.h) are supported in this implementation. This Compatibility Layer will be supported only as a short term solution to provide backward compatibility. **We recommend using the redesigned [coreMQTT library](https://github.com/FreeRTOS/coreMQTT/blob/master/README.md) for applications requiring long term support.**
```

Comment:
Is this accurate? "The MQTT Compatibility Layer provides backward compatibility from coreMQTT APIs to MQTT V2.x.x APIs <please provide a hyperlink>. We recommend that you use the coreMQTT library for optimized memory usage and modularity, but you can use the compatibility layer when you do not have the flexibility.

Reply:
That is correct. I will update the wording. Thanks